### ISSUE TITLE
[db-branch] routes

### ISSUE THREAD

**Issue by Vanshika-73** [LuciAkirami/express-rest-api#7]
will create routes for the operations to be performed and will test it through POSTMAN

**Comment by LuciAkirami** [comment]
Yup, and if your using GitHub Codespaces, you can install Thunderclient extension, that will allow you test the routes without needing to log in

**Comment by Vanshika-73** [comment]
okayy, but i have never used this. So, I prefer postman for testing. but you can go for it for testing purpose.

**Comment by LuciAkirami** [comment]
Yah you can use any of the tool for testing. The issue is yours
